src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = undefined) {
+    // Bubble Sort
+    for (let i = 0; i < this.length - 1; ++i) {
+      let swapped = false;
+
+      for (let j = 0; j < this.length - i - 1; ++j) {
+        // Convert values to strings by default
+        let compareResult = String(this[j]) > String(this[j + 1]);
+
+        if (compareFunction) {
+          compareResult = compareFunction(this[j], this[j + 1]);
+        }
```

Comment:
no needs to check as we have a default param value
```suggestion
```

Reply:
Done ✅